google-cloud-firestore/src/main/java/com/google/cloud/firestore/Query.java

```diff
@@ -1232,6 +1211,11 @@ public Query select(FieldPath... fieldPaths) {
     return new Query(rpcContext, newOptions.build());
   }
 
+  @Override
+  public boolean isRetryableWithCursor() {
```

Comment:
I don't think we want this as part of our API surface.

```suggestion
  boolean isRetryableWithCursor() {
```

Reply:
Great catch. Thanks.